tests/Type/PHPUnit/data/assert-function.php

```diff
@@ -20,15 +20,21 @@ public function doFoo($o): void
 		assertType(self::class, $o);
 	}
 
-	public function arrayHasNumericKey(array $a): void {
+	public function arrayHasNumericKey(array $a, \ArrayAccess $b): void {
 		assertArrayHasKey(0, $a);
 		assertType('array&hasOffset(0)', $a);
+
+		assertArrayHasKey(0, $b);
+		assertType('ArrayAccess', $b);
```

Comment:
I wanted to obtain this result but for some reasons it didn't work. However before my changes the result was `*NEVER*` so it's already an improvement.
```suggestion
		assertType('ArrayAccess&hasOffset(0)', $b);
```